index.html

```diff
@@ -33,27 +33,27 @@ <h3>I'am a Software Developer</h3>
           class="main-icons"
           src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Git-hub-icon.png"
           alt="git-hub icon"
-        >
+        />
         <img
           class="main-icons"
-          src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Icon - Linkedin.png"
+          src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio\Icons\Icon - Linkedin.png"
           alt="instagram icon"
-        >
+        />
         <img
           class="main-icons"
           src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Icont number 3.png"
           alt="new icon"
-        >
+        />
         <img
           class="main-icons"
           src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Twitter icon.png"
           alt="twitter icon"
-        >
+        />
         <img
           class="main-icons"
           src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Microverse Icon.png"
           alt="microverse icon"
-        >
+        />
```

Comment:
For your social icons your `src` paths should be like this:
```
<div class="container-icons">
        <img
          class="main-icons"
          src="./Icons/Git-hub-icon.png"
          alt="git-hub icon"
        />
        <img
          class="main-icons"
          src="./Icons/Icon - Linkedin.png"
          alt="instagram icon"
        />
        <img
          class="main-icons"
          src="./Icons/Icont number 3.png"
          alt="new icon"
        />
        <img
          class="main-icons"
          src="./Icons/Twitter icon.png"
          alt="twitter icon"
        />
        <img
          class="main-icons"
          src="./Icons/Microverse Icon.png"
          alt="microverse icon"
        />
      </div>
```

As well as your illustration container image:
```
    <div class="container-illustration">
      <img
        class="poligon-illustration"
        src="./Icons/Illustration.png"
        alt="Image of poligons"
      />
    </div>
```

Now your linter issues are solved and also your images are also displaying properly:
Before | After
------------- | -------------
![image](https://user-images.githubusercontent.com/38088818/164547920-9b4060bd-e086-4517-934c-885d7bae858c.png) | ![image](https://user-images.githubusercontent.com/38088818/164547812-4b055a7e-d700-4da2-89c1-864880768827.png)